proto.json

```diff
@@ -171,8 +171,11 @@
     "syft.messaging.message.CryptenInit": {
       "code": 50
     },
-    "syft.frameworks.torch.fl.dataset.BaseDataset": {
+    "torch.dtype": {
       "code": 51
+    },
+    "syft.frameworks.torch.fl.dataset.BaseDataset": {
+      "code": 52
```

Comment:
The code for BaseDataset should not change between versions.  Could you add your new type at the end?

Reply:
Done :)